recipes/nodejs_from_source.rb

```diff
@@ -24,9 +24,13 @@
 
 case node['platform_family']
 when 'rhel', 'fedora', 'amazon'
-  package %w(openssl-devel python3 tar)
+  package %w(openssl-devel python2 tar)
+  link '/usr/bin/python' do
+    to '/usr/bin/python2'
+    action :create
```

Comment:
This is the default action for `link`, so you can go ahead and remove it.

Reply:
Fixed.